### PR TITLE
Initialize directory structure with clients package

### PR DESCRIPTION
**Pull request by ryankwilliams**
# Change

This PR initializes the directory structure for the osde2e-common module. Included is the clients package providing the following clients:

- ocm
- openshift
- prometheus

For [SDCICD-1030](https://issues.redhat.com//browse/SDCICD-1030)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.DEFAULT_GOAL := lint
+
+format:
+	gofmt -w .
```
Comment (jbpratt) [inline Makefile:4]: can we drop the format target? gofumpt handles everything gofmt does plus more

**Comment by ryankwilliams** [comment]
/unhold

Good to go!

**Pull request merged**